cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec, *rspec.Spec) error {
+	fmt.Println("validating Readonly Paths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
This looks like host-mount-namespace access, but the read-only paths may only exist in the container mount namespace.  Is there code somewhere to enter the container mount namespace for these checks?